openshift/init_tenant.go

```diff
@@ -145,6 +144,7 @@ func do(config Config, callback Callback, username, usertoken string, templateVa
 // or default to the OOTB template included
 func loadTemplate(config Config, name string) ([]byte, error) {
 	teamVersion := config.TeamVersion
+	logCallback := config.LogCallback
```

Comment:
And the log interface needs to be more 'advanced' to handle the way the InitiTenet function use it. (as you will need that callback function since it does most of the work)